src/index.js

```diff
@@ -136,5 +136,22 @@ module.exports = {
     }
     return isHoliday;
   },
+  isAHolidayUTC(date = new Date(), federalReserveMode = false) {
```

Comment:
🙌 

Normally I'd have done it all in UTC in the first place, but there's a tickle in my brain about the date being consistently wrong when I was testing it.  In hindsight, it might be related to a Docker container.